frontend/js/src/settings/music-services/details/MusicServices.tsx

```diff
@@ -401,6 +556,91 @@ export default function MusicServices() {
           </div>
         </div>
 
+        <div className="card">
+          <div className="card-header">
+            <h3 className="card-title">Funkwhale</h3>
+          </div>
+          <div className="card-body">
+            <p>
+              Connect to your Funkwhale server to play music on ListenBrainz.
+            </p>
+            <div
+              className="alert alert-warning alert-dismissible fade show"
+              role="alert"
+            >
+              <strong>Important:</strong> You must be already logged into your
+              Funkwhale server before connecting it to ListenBrainz.
+              <button
+                type="button"
+                className="btn-close"
+                data-bs-dismiss="alert"
+                aria-label="Close"
+              />
+            </div>
```

Comment:
Let's hide the warning when connected (I recently did the same for LastFM)
```suggestion
            {permissions.funkwhale !== "listen" && (
              <div
                className="alert alert-warning alert-dismissible fade show"
                role="alert"
              >
                <strong>Important:</strong> You must be already logged into your
                Funkwhale server before connecting it to ListenBrainz.
                <button
                  type="button"
                  className="btn-close"
                  data-bs-dismiss="alert"
                  aria-label="Close"
                />
              </div>
            )}
```